radixdlt-java/src/main/java/com/radixdlt/client/application/identity/RadixIdentities.java

```diff
@@ -121,7 +121,7 @@ public static RadixIdentity loadOrCreateEncryptedFile(String filePath, String pa
 			throws IOException, PrivateKeyException, PublicKeyException, KeyStoreException {
 		var keyFile = new File(filePath);
 		var keyStore = RadixKeyStore.fromFile(keyFile, password.toCharArray(), !keyFile.exists());
-		var keyPair = keyStore.readKeyPair(keyName, !keyFile.exists());
+		var keyPair = keyStore.readKeyPair(keyName, false);
```

Comment:
should we change the name of this method, since it no longer creates?

Reply:
Sorry, it's a leftover from experiments. Fixed now.